src/dune_rpc_impl/decl.ml

```diff
@@ -52,14 +46,14 @@ module Status = struct
   let decl = Decl.Request.make ~method_:"status" ~generations:[ v1 ]
 end
 
-module Build = struct
+module Wait = struct
   let v1 =
-    Decl.Request.make_current_gen ~req:(Conv.list Conv.string)
-      ~resp:Build_outcome.sexp ~version:1
+    Decl.Request.make_current_gen ~req:Conv.unit ~resp:Build_outcome.sexp
+      ~version:1
```

Comment:
Do we need to bump the version here, since we're changing the interface? I don't actually think there are any clients, but we probably want to be sure just in case

Reply:
This request was only added to support the test suite. Since it is defined in `dune_rpc_impl`, I would hope that no client ended up using it.

Reply:
Plus it requires starting Dune in a special mode, so it's quite likely that it's ununsed.